skymp5-server/cpp/papyrus_vm_lib/Reader.cpp

```diff
@@ -41,13 +41,13 @@ Reader::Reader(std::vector<std::string> vectorPath)
   }
 }
 
-Reader::Reader(std::vector<std::vector<uint8_t>> pexVector)
+Reader::Reader(const std::vector<std::vector<uint8_t>>& pexVector)
 {
   for (auto& pex : pexVector)
     CreateScriptStructure(pex);
 }
 
-void Reader::CreateScriptStructure(std::vector<uint8_t> arrayBytes)
+void Reader::CreateScriptStructure(const std::vector<uint8_t>& arrayBytes)
 {
   this->arrayBytes = arrayBytes;
```

Comment:
Actually could say something like this to allow moving
```suggestion
void Reader::CreateScriptStructure(std::vector<uint8_t> arrayBytes)
{
  this->arrayBytes = std::move(arrayBytes);
```